packages/j/jsonpath-dchester.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "jsonpath",
+  "filename": "jsonpath.min.js",
+  "description": "Query JavaScript objects with JSONPath expressions. Robust / safe JSONPath engine for Node.js.",
+  "homepage": "https://github.com/dchester/jsonpath#readme",
+  "keywords": [
+    "jsonpath",
+    "dchester",
+    "JSONPath",
+    "json-path",
+    "object",
+    "traversal",
+    "json",
+    "path",
+    "data structures",
+    "popular"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dchester/jsonpath.git"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "npm",
+    "target": "jsonpath",
+    "fileMap": [
+      {
+        "basePath": ".",
+        "files": [
+          "**/jsonpath.js",
+          "**/jsonpath.min.js"
```

```suggestion
          "jsonpath?(.min).js"
```
We try to avoid using globstars wherever possible, as it can lead to including unexpected files in the CDN. It looks like these files are at the root of the package, so we don't need to specify any directory/globstar. (We can also simplify this into a single glob)

---

Changed